### ISSUE TITLE
Provide link to Insider-specific docs

### ISSUE THREAD

**Issue by cfjedimaster** [Microsoft/vscode#2724]
I raised this on Twitter, but just to be clear, there should be an easy way for someone using the Insider build to get to release notes/specifics about the Insider Build. Currently the menu drop downs for Documentation/Release Notes go to the 'real' links for the released version. But there is no way for me to find out what's new in this release. The blog post (https://blogs.msdn.microsoft.com/vscode/2016/02/01/introducing-insiders-builds/) about this release *does* tell you where to go, but if you don't have that post bookmarked (or the link), then you're kind of lost.

Basically - provide an easy way for someone running the Insider build to figure out what's new/updated/bugged/etc with it.

**Comment by egamma** [comment]
@chrisdias I'm seeing the same behaviour, but I thought you did some fwlink magic for the insiders build?

**Comment by chrisdias** [comment]
The problem is the Insiders build used the same link to the release notes as the stable build.  if i changed the FWLink target then the current stable build would be pointing to the new release notes.  

i created a new FWLink for the Insiders build for the release notes and updated the product.json to use this new FWLink. This means that then next Insiders build we distribute should have the correct endpoint.
